test/e2e/kube_state_metrics_test.go

```diff
@@ -15,6 +15,12 @@
 package e2e
 
 import (
+	v1 "k8s.io/api/autoscaling/v1"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	vpav1 "k8s.io/autoscaler/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1"
+	"path"
```

Comment:
(nit) can you rearrange the import list to have the stdlib packages first and separated from the rest?